dir.props

```diff
@@ -252,8 +252,6 @@
     <BuildConfigurationRefPath Condition="$(_bc_TargetGroup.EndsWith('aot'))">$(RefRootPath)$(_bc_TargetGroup.TrimEnd('aot'))/</BuildConfigurationRefPath>
     <RefPath>$(RefRootPath)$(TargetGroup)/</RefPath>
     <RefPath Condition="$(TargetGroup.EndsWith('aot'))">$(RefRootPath)$(TargetGroup.TrimEnd('aot'))/</RefPath>
-    <!-- tests don't yet set appropriate configurations -->
-    <RefPath Condition="'$(IsTestProject)' == 'true'">$(BuildConfigurationRefPath)</RefPath>
```

Comment:
no this is just getting the test projects compile against the configuration with netcoreapp. we'll need to go and fix the projects more for other verticals (netfx, UAP...etc.)